src/manage/commands.py

```diff
@@ -476,73 +514,62 @@ def execute(self):
 
     @classmethod
     def usage_text_lines(cls):
-        usage_docs = [
-            (f"    {EXE_NAME} -V:!B!<TAG>!W!",
-             "Launch runtime identified by !B!<TAG>!W!, which should include the " +
-             "company name if not !B!PythonCore!W!. Regular Python options may " +
-             "follow this option."),
-            (f"    {EXE_NAME} -!B!<VERSION>!W!",
-             r"Equivalent to -V:PythonCore\!B!<VERSION>!W!. The version must " +
-             "begin with the digit 3, platform overrides are permitted, " +
-             "and regular Python options may follow." +
-             (" !G!py -3!W! is the equivalent of the !G!python3!W! command." if EXE_NAME == "py" else "")),
-            (f"    {EXE_NAME} !B!<COMMAND>!W!",
-             "Run a specific command (see list below)."),
-        ]
-
-        usage_ljust = max(len(logging.strip_colour(i[0])) for i in usage_docs)
+        if EXE_NAME.casefold() in ("py".casefold(), "pyw".casefold()):
+            usage_docs = PY_USAGE_DOCS
+        else:
+            usage_docs = PYMANAGER_USAGE_DOCS
+
+        usage_docs.extend(
+            [
+                (
+                    f"{EXE_NAME} " + getattr(COMMANDS[cmd], "USAGE_LINE", cmd),
+                    getattr(COMMANDS[cmd], "HELP_LINE", "")
+                )
+                for cmd in sorted(COMMANDS)
+                if cmd[:1].isalpha()
+            ]
+        )
```

Comment:
Slightly less indented
```suggestion
        usage_docs += [
            (
                f"{EXE_NAME} " + getattr(COMMANDS[cmd], "USAGE_LINE", cmd),
                getattr(COMMANDS[cmd], "HELP_LINE", "")
            )
            for cmd in sorted(COMMANDS)
            if cmd[:1].isalpha()
        ]
```

Reply:
Will do, but it actually needs to be `usage_docs = usage_docs + ...` to avoid modifying the global (not that it really matters here, but it will in tests)